### ISSUE TITLE
Something is not working when editing Spanish Script text

### ISSUE THREAD

**Issue by gutialv23** [kwsch/pkNX#114]
Exporting the default text and importing it again (this should do nothing in practice) breaks the texts. After that you will find things like "Gym Leader Superpotion" instead of "Gym Leader Cathy" (but in Spanish, of course)

I did not try with other languages.

**Comment by kwsch** [comment]
Can you export the text file again and compare it to what you imported?

**Comment by gutialv23** [comment]
Yes, I have already done that and, as you supposed, they are different.
Look:
[script_default.txt](https://github.com/kwsch/pkNX/files/5659251/script_default.txt)
[script_default_reimported.txt](https://github.com/kwsch/pkNX/files/5659252/script_default_reimported.txt)

**Comment by gutialv23** [comment]
I have tried to just Edit and Save using the Text Editor in PkNX and all those changes also happen in every empty line so... I think that the problem is not in the import/export part. The bug may happen while saving changes?

**Comment by kwsch** [comment]
Empty lines getting replaced with the placeholder variable and leading spaces getting removed is not the cause of the behavior you're describing.

**Comment by gutialv23** [comment]
Ah perfect, I supposed it could be that. I have not seen any other changes

**Comment by kwsch** [comment]
Closed by https://github.com/kwsch/pkNX/pull/380
